lib/rucio/web/ui/static/list_lifetime_exceptions.js

```diff
@@ -0,0 +1,108 @@
+/* Copyright European Organization for Nuclear Research (CERN)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Authors:
+ * - Cedric Serfon, <cedric.serfon@cern.ch>, 2018
+ */
+
+function format ( d ) {
+    str = '<table cellpadding="5" cellspacing="0" border="0" style="padding-left:50px;">';
+    for (var cnt=0; cnt<d.datasets.length;cnt++){
+        str += '<tr>';
+        if(cnt == 0){
+            str += '<td width="200">Dataset list:</td>';
+        }
+        else{
+            str += '<td width="200"></td>';
+        }
+        var scope = d.datasets[cnt].split(':')[0];
+        var name = d.datasets[cnt].split(':')[1];
+        str += '<td><a href="/did?scope=' + scope + '&name=' + name + '">' + d.datasets[cnt] + '</a></td>';
+        str += '</tr>';
+    }
+    str += '</table>';
+    return str
+}
+
+function formatBytes(bytes){
+   if(bytes == 0) return '0 Bytes';
+   sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB'],
+   i = Math.floor(Math.log10(bytes) / 3);
+   return parseFloat(bytes / Math.pow(1000, i)).toFixed(2) + ' ' + sizes[i];
+}
+
+$(document).ready(function(){
+    $('#loader').html('<div class="row"><div class="large-1 large-centered columns"><img src="/media/spinner.gif"></div></div>');
+    r.list_lifetime_exceptions({
+         success: function(data) {
+            var dict = {};
+            data = data.split('\n');
```

Comment:
Not a big thing but maybe a bit cleaner: There is already a `parse_json_stream` function in rucio.js that is used for the other json streaming calls. That will take care of the splitting, checking for empty data and parsing the json for you.